index.html

```diff
@@ -1960,6 +1963,12 @@ <h4 id="maturity-levels">6.1.2 Maturity Levels</h4>
           abandoned without producing a Recommendation.</dd>
       </dl>
 
+      <p>Advancement to a new maturity level must be made based on the technical maturity of the work
```

Comment:
Seems like we're picking on "scheduling considerations" to the exclusion of other potential motivations for pressing ahead. Suggest rewording to:

> The decision to advance to a new maturity level must be made solely based on the technical maturity of the work. 
> Note: should faster advancement to meet scheduling considerations be desired, this may be achieved by reducing the scope of the technical report to a subset that is adequately mature and deferring
> less stable features to other technical reports.

Reply:
I like your suggestion. Applying it.